go/x509/root_darwin.go

```diff
@@ -13,14 +13,14 @@ package x509
 #include <CoreFoundation/CoreFoundation.h>
 #include <Security/Security.h>
 
-// FetchPEMRoots fetches the system's list of trusted X.509 root certificates.
+// _FetchPEMRoots fetches the system's list of trusted X.509 root certificates.
```

Comment:
Name updated so as to not conflict with the `FetchPEMRoots` defined in the standard library.